docs/pages/guides/using-eslint.mdx

```diff
@@ -142,15 +142,29 @@ To install Prettier in your project:
 <Terminal
   cmd={[
     '# For other package managers',
-    '$ npx expo install -- --save-dev prettier',
+    '$ npx expo install -- --save-dev prettier eslint-config-prettier eslint-plugin-prettier',
     '',
     '# For yarn only',
-    '$ yarn add -D prettier',
+    '$ yarn add -D prettier eslint-config-prettier eslint-plugin-prettier',
   ]}
 />
 
 ### Setup
 
+To integrate with ESlint, update your `.eslintrc.js`:
+
+```js .eslintrc.js
+module.exports = {
+  extends: ["expo", "prettier"],
+  plugins: ["prettier"],
+  rules: {
+    "prettier/prettier": "error",
+  },
+};
+```
+
+Now when you running `npx expo lint`, anything not alighned with prettier formatting will be caught as an error. Side note: you may want to do `"prettier/prettier": "warn"` instead if you prefer these to be warnings instead of errors.
```

Comment:
```suggestion
Now, when you run `npx expo lint`, anything that is not aligned with Prettier formatting will be caught as an error. 

> **Note:** In the above configuration, you can use `"prettier/prettier": "warn"` if you prefer these formatting issues as warnings instead of errors.
```

- Fix typos in the first line
- Improve verbiage for readability in the first line
- Let's make the side note a callout